examples/langgraph-memanto/README.md

```
@@ -0,0 +1,108 @@
+
+--- Query 2 ---
+Input: Alice reported a billing discrepancy...
+
+Classification: remember
+Tool result: Memory stored successfully. ID: mem-def456...
+Response: I've recorded the billing issue...
+```
+
+## Session 2 Output (different process, next day)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)